lib/cmdledit.g

```diff
@@ -262,7 +262,7 @@ GAPInfo.UseReadline := true;
 
 
 if not IsBound(GAPInfo.CommandLineEditFunctions) then
-  GAPInfo.CommandLineEditFunctions := rec(
```

Comment:
So... are we OK with just introducing uses of `AtomicRecord` into the code base? Of course it is essentially a NOP in regular GAP (here, it will just return its argument).
I personally don't mind, but perhaps @frankluebeck does?